app/src/main/java/edu/grinnell/kdic/RadioService.java

```diff
@@ -225,21 +200,21 @@ public void setRunOnStreamPrepared(final Runnable runOnStreamPrepared) {
         this.runOnStreamPrepared = runOnStreamPrepared;
     }
 
+    /*
+        Plays the currently loaded stream. If stream is not loaded, load stream and play.
+    */
     public void play() {
```

Comment:
Should this method be `public`? Check if it is called elsewhere, else change it to `private`